fecgov/fec-cms
# Add most recent Form 1 to committee profile pages

## Summary (required)

- Resolves #2964

Add most recent Form 1 to committee profile page "About this Committee" tab, add test coverage, move "most recent" logic out of template

Note: Candidate profile page shows most recent regardless of cycle, so for now that's what we're doing with committee profile pages. 

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Committee profile page "About this Committee" tab, Candidate profile "About this candidate" tab

## Screenshots

## Before
![Screen Shot 2019-10-17 at 9.57.00 AM.png](https://images.zenhubusercontent.com/59a579f28f62dc7798c47359/a5077d95-c6f1-4f11-a314-784d8fe69b7e)

## After
![Screen Shot 2019-10-25 at 10.59.26 AM.png](https://images.zenhubusercontent.com/59a579f28f62dc7798c47359/eb947db1-e856-4e95-bea2-3a40b96e254f)

## How to test
-  run this branch, check out http://localhost:8000/data/committee/C00693234/?tab=about-committee&cycle=2020
- this **won't show up** for non-committee filers, example: http://localhost:8000/data/committee/C90010661/?tab=about-committee
- Candidate page: http://localhost:8000/data/candidate/H8CA05035/?tab=about-candidate&cycle=2018
